packages/manager/modules/manager-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ovh-ux/manager-components",
-  "version": "1.19.5",
+  "version": "1.20.0",
```

Comment:
**nitpick:** No need to update the version manually. It will be automatically handled based on the type of the commit.